Model/Client.php

```diff
@@ -36,7 +36,7 @@ class Client
      */
     private $active = true;
 
-    public function __construct(string $identifier, string $secret)
+    public function __construct(string $identifier, ?string $secret)
```

Comment:
@X-Coder264 Should we add a default null value here?

Reply:
@HypeMC I don't think there is a need for that. The secret was a required argument even before this PR so we are explicitly passing the argument in all code paths.